packages/nextjs/src/app-beta/currentUser.ts

```diff
@@ -1,7 +1,9 @@
+import type { User } from '@clerk/backend';
+
 import { auth } from './auth';
 import { clerkClient } from './clerkClient';
 
-export function currentUser() {
+export async function currentUser(): Promise<User | null> {
```

Comment:
❓ Is this a breaking change?

Reply:
Nope.

Reply:
Oh, it's in the `app-beta` directory.

Reply:
TS would still require an await there anyway, as Promise was already in the signature